src/NUnitFramework/framework/Diagnostics/ProgressTraceListener.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
+
+using System;
+using System.Diagnostics;
+#if !NET6_0_OR_GREATER
+using System.Security.Permissions;
+#endif
+
+namespace NUnit.Framework.Diagnostics
+{
+    /// <summary><para>
+    /// The ProgressTraceListener class allows directing tracing or
+    /// debugging output to <see cref="TestContext.Progress"/>.
+    /// </para><para>
+    /// To activate, place the following snippet into the one-time
+    /// set-up method of either a test's fixture or the set-up
+    /// fixture of a project:
+    /// <c>
+    /// System.Trace.Listeners.Add(new ProgressTraceListener());
+    /// </c>
+    /// </para><para>
+    /// Make sure to only add a listener once, e.g.:
+    /// <c>
+    /// if (!System.Trace.Listeners.OfType&lt;ProgressTraceListener&gt;().Any())
+    ///     System.Trace.Listeners.Add(new ProgressTraceListener());
+    /// </c>
+    /// </para><para>
+    /// Or again remove in the one-time tear-down, e.g.:
+    /// <c>
+    /// System.Trace.Listeners.Remove(_progressTraceListener);
```

Comment:
```suggestion
    /// System.Trace.Listeners.Remove(_progressTraceListener);
    _progressTraceListener.Dispose();
```

The listener is `IDisposable`.

Reply:
According to https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.textwritertracelistener.dispose, `Close()` should be used instead of `Dispose()`.